pkg/broker/receiver.go

```diff
@@ -37,6 +37,10 @@ import (
 
 const (
 	writeTimeout = 1 * time.Minute
+	// TimeInFlightMetadataName is used to access the metadata stored on a
+	// CloudEvent to measure the time difference between when an event is
+	// received and when it is dispatched to the trigger function.
+	TimeInFlightMetadataName = "kn00timeinflight"
```

Comment:
What's the reasoning for choosing `kn00` as the prefix instead of `knative` or `knative-`?

Reply:
There's a 20-character limit. This is presently 16. This was also the recommended name on the issue itself. I didn't think to look for other examples of this being used here so I didn't notice if there was precedent for knative or knative- as the prefix.

Reply:
I didn't know about the 20 character recommendation, thanks!